Quantum-Control-Applications/Superconducting/Kerr-Cat SNAIL-mon/Use Case 2 - QuLAB - Kerr-Cat/readme.md

```diff
@@ -0,0 +1,200 @@
+of the single-photon "Fock qubit" shown for comparison with **a**. **c**, Energy dependence of equation (1) on classical
+phase-space coordinates Re(a) and Im(a) for squeezing drive amplitudes $\epsilon\_{2}/2\pi$ = 17.75 MHz (left)
+and $\epsilon\_{2}$ = 0 (right) with a sketch showing the adiabatic ramp of the drive over a time $\tau\_{ramp} >> 1/2K$. Black
+lies are constant energy contours. The quadrature expectation values of the |$\pm X$ $\>$-states from **a**, **b** are 
+indicated by their respective markers.
+
+## Cat quadrature readout
+
+Cat quadrature readout (cqr) is a Quantum non-Demolition (QND) technique to readout the Kerr-cat qubit.
+Thanks to the three-wave mixing of the SNAIL, a cqr pulse sent to it at $\omega_{resonator}$ - $\omega_{s}/2$ creates a
+coherent drive in the readout resonator that we can measure. Figure 2 and 3 show the IQ-blobs
+of the coherent states |$\alpha$ $>$ and |-$\alpha$ $>$ and the pulse sequence, respectively.
+
+![Figure_2a_IQblobs](Figure_2a_IQblobs.png)
+
+**Figure 2**, IQ blobs of the coherent states |$\alpha$ $>$ and |$-\alpha$ $>$ measured with cat quadrature readout (cqr).
+
+<img alt="Fig2_pulse_sequence" height="153" src="Fig2_Pulse_sequence.png" width="350"/>
+
+**Figure 3**, cartoon of the pulse sequence to readout the coherent states with cqr.
+
+The main lines of QUA code that produce Fig. 2 are shown below. The QUA code that reproduces the data is `cqr_blobs_yale.py`.
+
+```python
+        play('on', 'squeeze_switch', duration=int((4e6+1e3)//4))
+        play('ftc_rise', 'squeeze_rise')
+        align('squeeze_rise', 'squeeze_drive')
+        play('cw', 'squeeze_drive', duration=int(4e6//4))
+        align('squeeze_drive', 'squeeze_fall')
+        play('ftc_fall', 'squeeze_fall')
+        
+        wait(cooldown_time, 'cqr_drive', 'resonator')
+                
+        with for_(i, 0, i < 1, i+1):
+            play('on', 'cqr_switch', duration=(cqr_len//4))
+            play('cqr', 'cqr_drive')
+            play('on', 'SPC_pump', duration=(passive_len//4))
+            measure('passive_readout', 'resonator', None, demod.full('rotated_cos', I, 'out1'), demod.full('rotated_sin', Q, 'out1'))
+            save(I, I_st)
+            save(Q, Q_st)
+```
+
+## Coherence times
+
+The Kerr-cat Hamiltonian is resilient to phase-flips because the coherent states |$\alpha$ $>$ and
+|$-\alpha$ $>$ are separated by an energy barrier. We can use cqr to infer the lifetimes of the
+|$\alpha$ $>$ and |$-\alpha$ $>$ states.
+
+The pulse sequence, see Fig. 5, to measure the lifetime is the following: (i) turn-on the squeeze-drive,
+(ii) wait for a cooldown-time, (iii) apply a cqr pulse and do state discrimination, (iv) wait for a variable
+delay, and (v) apply a second cqr pulse. This pulse sequence is repeated N times to get good SNR.
+
+The experimental data can be seen in Fig. 4, and by fitting it to single exponential decay we find
+$\tau_{\alpha} = 163 \mu s$ and $\tau_{-\alpha} = 164 \mu s$.
```

Comment:
Can we compare these values to the coherence time of the standard fock states to emphasize the improvement?